From dfd1a2609ffd4ae01ccf9ddb8cff83325e8767ce Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Thu, 4 Oct 2018 17:05:03 -0500
Subject: [PATCH 3/6] Update default crio.conf file for Clear Linux

Update paths to policy.json, seccomp.json, conmon and cni plugins
to reflect where they are intended to be used from Clear Linux.
Add docker.io as image registry. Set overlay as default storage
driver.

Signed-off-by: Jose Lamego <jose.a.lamego@intel.com>
---
--- a/Makefile	2019-11-06 22:31:31.705839929 +0000
+++ b/Makefile	2019-11-06 22:32:07.944624208 +0000
@@ -141,6 +141,13 @@
 
 crio.conf: bin/crio
	./bin/crio --config="" $(CONF_OVERRIDES) config  > crio.conf
+	sed -i 's|^\(conmon = \).*|\1"/usr/libexec/crio/conmon"|' crio.conf
+	sed -i 's|^\(seccomp_profile = \).*|\1"/usr/share/defaults/crio/seccomp.json"|' crio.conf
+	sed -i "s|^#\(registries = \[\).*|\1\n\t'docker.io',\n\]|" crio.conf
+	sed -i 's|^\(signature_policy = \).*|\1"/usr/share/defaults/crio/policy.json"|' crio.conf
+	sed -i 's|^#\(storage_driver = \).*|\1"overlay"|' crio.conf
+	sed -i 's|^\(plugin_dirs = \[\).*|\1\n\t"/usr/libexec/cni",|' crio.conf
+	sed -i 's|^\(cgroup_manager = \).*|\1"systemd"|' crio.conf
 
 release-note: ${RELEASE_TOOL}
	${RELEASE_TOOL} -n $(release)
